resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -507,6 +512,12 @@ protected void doResolve(String inetHost,
             return;
         }
 
+        if (inetHost.isEmpty()) {
```

Comment:
@Scottmitch maybe we even should just move the empty and null handling to `SimpleNameResolver` ?